sdk/src/main/scala/com/horizen/validation/ConsensusValidator.scala

```diff
@@ -34,6 +34,9 @@ class ConsensusValidator extends HistoryBlockValidator with ScorexLogging {
 
 
   private def validateNonGenesisBlock(verifiedBlock: SidechainBlock, history: SidechainHistory): Unit = {
+    val parentBlockInfo: SidechainBlockInfo = history.blockInfoById(verifiedBlock.parentId)
```

Comment:
something is wrong here after merge. Please, look deeply inside `ourboros_vrf_all_nonce` branch code